releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,29 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 8.1.0 - February 13th, 2025
+### Enhancements
+* Added the ability to monitor all selected key-values for refresh with the following new API. [#536](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/536)
```

Comment:
Can you make sure that the issue description is up to date with any design changes that came up during code reviews

Reply:
Updated from what I can tell, I'll double check the PR again